pages/index.html

```diff
@@ -1,12 +1,26 @@
-<!DOCTYPE html>
-<html lang="en">
+<!DOCTYPE html >
+<html lang="en" class="Index">
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="../style.css">
   <title>Document</title>
 </head>
 <body>
+  <header>
+  <h1> Welcome to Diana's Portfolio </h1>
+  </header>
+  <a href="/pages/portfolio.html"><img class= "Dianaport" src="/images/Supermassiveblackhole.png"></a>
```

Comment:
For accessibility your `img` elements should have an `alt` attribute.

```suggestion
  <a href="/pages/portfolio.html"><img class= "Dianaport" src="/images/Supermassiveblackhole.png" alt="supermassive black hole" /></a>
```